### DISCUSSION TITLE
5' and 3' GEX integration

### DISCUSSION THREAD

**Discussion by tdfy** [satijalab/seurat#4324] [Q&A] [answered]
Hello, 

I am interested in integrating 5' and 3' 10x gene expression libraries. Presently, we use SCTransform for our current workflow. Are there any additional considerations for batch correction given the different chemistries in library construction? Thanks. 

-Todd

**Comment by timoast** [comment, marked as answer]
No, you should be able to follow the integration vignettes shown on the Seurat website
